tests/067-time.buzz

```diff
@@ -0,0 +1,45 @@
+import "std";
+import "time";
+
+test "Time std lib" {
+    const now = DateTime.now();
+
+    const time = DateTime{
+        gmtOffset = 0,
+        hour = 9,
+        isdst = 1,
+        minute = 38,
+        month = 9,
+        monthDay = 5,
+        second = 55,
+        timeZone = "",
+        weekDay = 4,
+        year = 123,
+        yearDay = 277,
+    };
+
+    const format = "The time is %Y-%m-%d %H:%M:%S";
+    const formatted = time.format(format);
+    assert(
+        formatted == "The time is 2023-10-05 09:38:55",
+        message: "Could format time"
+    );
+
+    assert(time.timestamp() == 1696491535, message: "Could use timestamp");
```

Comment:
```suggestion
    assert(time.timestamp() == 1696523935, message: "Could use timestamp");
```

I don't know if this is _correct_ but it's what I'm seeing on Mac if I `dump(time.timestamp())` before this line.

Reply:
Oh if it's timezone related... I'm in the Pacific timezone